auth/src/main/java/org/jboss/pnc/auth/JAASLoggedInUser.java

```diff
@@ -39,7 +39,7 @@ public JAASLoggedInUser(HttpServletRequest httpServletRequest) {
 
     @Override
     public String getEmail() {
-        return "Email N/A (" + getUserName() + ")";
+        return "example@example.com";
```

Comment:
Yes, I changed it due to the constraint and removing it is not the right way, I think.

I don't understand the A):
How can I read it from the User entity, when the entity is meant to be created based on what's obtained from the JAASAuthenticator.

Currently there is no e-mail written to the DB, so I can change it to smth like:
getUserName() + "@example.com"
or 
getUserName() + "@unknown.com"
to preserve the information there and to have the correct e-mail address format.